tests/tabular/checks/train_test_validation/multivariate_drift_test.py

```diff
@@ -168,4 +220,58 @@ def test_over_255_categories_in_column():
 
     # Assert
     # we only care that it runs
-    assert_that(result.value['domain_classifier_auc'])
+    assert_that(result.value["domain_classifier_auc"])
+
+
+def test_raise_zero_timeout(drifted_data):
```

Comment:
These last two tests are the only changes that should be kept here.

Reply:
Right, so I redid the branch, I hope it's still working